web/static/main.css

```diff
@@ -139,8 +139,16 @@ main:after {
        clear: both;
 }
 
-.entryItem {
+.entryItem-odd {
+       float: left;
+       display: table-column;
+       width: 200px;
+       background: #d4d4d4      
+}
+
+.entryItem-even {
        float: left;
        display: table-column;
        width: 200px;      
-}
\ No newline at end of file
+       background: #ccc
```

Comment:
Add a semicolon. It works without a semicolon, but semicolons are much, much better.